hack/coredump.sh

```diff
@@ -0,0 +1,74 @@
+#!/bin/bash
```

Comment:
maybe this should be in cluster/ instead?

also, logdump is probably a better name than coredump? or probably `log-dump.sh` given the naming convention in cluster/.